src/Sulu/Bundle/WebsiteBundle/Routing/PortalLoader.php

```diff
@@ -12,32 +12,48 @@
 namespace Sulu\Bundle\WebsiteBundle\Routing;
 
 use Sulu\Component\Webspace\Manager\WebspaceManagerInterface;
-use Symfony\Component\Config\Loader\Loader;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\Config\FileLocatorInterface;
+use Symfony\Component\Config\Loader\FileLoader;
 use Symfony\Component\Routing\Route;
 use Symfony\Component\Routing\RouteCollection;
 
 /**
  * This route loader is responsible for loading routes from a routing file, and adjust the routes in a way, so that
  * there will be an own route for every portal registered.
+ *
+ * @internal This class is internal and should not be instantiated or extended by any application or library.
  */
-class PortalLoader extends Loader
+class PortalLoader extends FileLoader
 {
     /**
      * @var WebspaceManagerInterface
      */
     private $webspaceManager;
 
-    public function __construct(WebspaceManagerInterface $webspaceManager)
-    {
+    public function __construct(
+        WebspaceManagerInterface $webspaceManager,
+        ?FileLocatorInterface $fileLocator = null
```

Comment:
this is not a bc break this way, right? should we target `2.1`?

Reply:
yes this should not be a bc break this way